schema/EAR_MEMO.json

```diff
@@ -2,38 +2,32 @@
     "$schema": "https://json-schema.org/draft-07/schema#",
     "$id": "https://github.com/fecgov/fecfile-validate/blob/main/schema/EAR_MEMO.json",
     "version": "v0.0.0.0",
-    "title": "FEC Earmark Receipt (11a - Memo)",
-    "description": "Earmark Receipt-Memo (11a)",
+    "title": "FEC Other Receipts",
+    "description": "SCHEDULE A - ITEMIZED RECEIPTS-Line 17 Other Receipts",
     "type": "object",
     "required": [
         "form_type",
         "filer_committee_id_number",
         "transaction_type_identifier",
         "transaction_id",
+        "back_reference_tran_id_number",
+        "back_reference_sched_name",
         "entity_type",
-        "contributor_organization_name",
```

Comment:
these fields are supposed to be in EARMARK_MEMO if we're following group G https://docs.google.com/spreadsheets/d/1gdcvYQBFMeCGt-1NfVuaSTsB0sB-dl2N/edit#gid=325373431

Reply:
The JSON file has been synced up wtih the spec spreadsheet which corrected this among other issues